lib/ansible/modules/validate_argument_spec.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Red Hat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
`ANSIBLE_METADATA` has been deprecated in https://github.com/ansible/ansible/pull/69454, is it needed here? If so we should document the purpose.

Reply:
Ah, nope. Good catch. This file was created way back in the before times.